test/documentation/bio-doxygen.cmake

```diff
@@ -0,0 +1,74 @@
+cmake_minimum_required (VERSION 3.7)
```

Comment:
I am not sure if we can also use SeqAn3's version of this file, but it would be nice.

Reply:
I don't think. this file defines

```
set (BIO_DOXYGEN_SOURCE_DIR "${BIO_CLONE_DIR}")
set (BIO_DOXYFILE_IN ${BIO_DOXYGEN_INPUT_DIR}/bio_doxygen_cfg.in)
```
The first is important to find all header files to get the documentation from.
The second is to get the doxygen config. Only if you never want to change the doxygen config for this project (which is already violated when we want the EXPAND macro to work) could we copy this from seqan.